tests/nosetests/pyanaconda_tests/module_payload_live_os_test.py

```diff
@@ -26,6 +26,9 @@
 from pyanaconda.dbus.typing import get_native
 from pyanaconda.modules.common.constants.objects import LIVE_OS_HANDLER
 from pyanaconda.modules.common.structures.storage import DeviceData
+
+from pyanaconda.modules.payload.shared.initialization import PrepareSystemForInstallation
+
```

Comment:
Please, remove the empty lines between imports.

Reply:
Fixed.